Project_home/css/style.css

```
@@ -131,12 +106,12 @@ header ul li a {
     align-items: center;
     margin: 158px auto 0;
 }
-.date, .poly, .doctor {
+.fix_1, .fix_2, .fix_3 {
```

Оскільки стилі для всіх елементів однакові, можете використати один клас для всіх.
```suggestion
.fix {
```